### PR TITLE
fixing formatting helps with #1367

### PR DESCRIPTION
**Pull request by jasonwilliams**
Adding to FAQ for anyone else with issue
https://github.com/rust-lang/rustup.rs/issues/1367

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -724,6 +724,11 @@ scanning in the background. Disable antivirus scanner and try again.
 [multirust]: https://github.com/brson/multirust
 [Diggory Blake]: https://github.com/Diggsey
 
+### I get "error: could not remove 'rustup-bin' file: 'C:\Users\USER\.cargo\bin\rustup.exe'"
```
Comment (tesuji) [inline README.md]: Might need `\\.cargo` as markdown escape it somehow.

Comment (kinnison) [reply README.md]: Looking at the render, it looks okay.

**Review by kinnison — APPROVED** [review]
This seems like a good FAQ to have documented.

Thank you for your contribution.

File: README.md (hunk 1 above)
Comment (jasonwilliams) [reply README.md]: adding the extra \

**Review by kinnison — APPROVED** [review]
Aah yes, I see what went wrong, okay cool.

**Comment by kinnison** [comment]
Actually it'd be nice if you could squash the two commits into one -- Do you know how to do that?

**Comment by jasonwilliams** [comment]
Yep, done @kinnison

**Comment by kinnison** [comment]
Thanks @jasonwilliams

**Pull request merged**